extensions/amp-story/1.0/test/test-animation.js

```diff
@@ -542,7 +542,9 @@ describes.realWin('amp-story animations', {}, (env) => {
         `,
         ampdoc
       );
-      return expect(() => animationManager.applyFirstFrame()).to.throw();
+      return allowConsoleError(() =>
```

Comment:
Double checking: it throws but doesn't prevent page navigation like on the example story?

Reply:
This is an unrelated test, this is for the preset name (`animate-in=`) which is handled the same as before. (Test started to complain without `allowConsoleError` for some reason).

Would we like to just warn when the preset name is invalid?